es-models/case_centric/case_centric.mapping.yaml

```diff
@@ -531,6 +531,12 @@ properties:
         type: boolean
       state:
         type: keyword
+  samples:
+    properties:
+      sample_type:
+        type: keyword
+      sample_type_id:
```

Comment:
Sample_type looks good here; I checked out the other field in the dictinoary and the data; We do not need sample_type_id so would remove that